packages/cardano-services-client/package.json

```diff
@@ -69,6 +69,7 @@
   "dependencies": {
     "@cardano-sdk/core": "workspace:~",
     "@cardano-sdk/util": "workspace:~",
+    "@koralabs/handles-public-api-interfaces": "1.6.6",
```

Comment:
This is only used for types that are not exposed (internal usage only), so it should probably be under `devDependencies`

Reply:
It is imported in the src/ folder, so if it's moved to the `devDependencies` the import won't work, and it will appear as listed on the wrong list.